src/libs/ajax.js

```diff
@@ -880,7 +880,7 @@ export const ToS = {
 
 export const Translate = {
   translate: async (body) => {
-    const url = `${await getOntologyUrl()}/translate/paragraph`;
+    const url = `https://consent-ontology.dsde-dev.broadinstitute.org/translate/paragraph`;
```

Comment:
`await getOntologyUrl()` was returning `https://consent-ontology.dsde-dev.broadinstitute.org/` (with the slash on the end of the string), but other urls in this ajax file looked like `${getOntologyUrl()}/autocomplete`, and when following that standard, the translate URL turned into

`https://consent-ontology.dsde-dev.broadinstitute.org//translate/paragraph` which would error out (because of the double slash)

Not sure if it's safe to remove that slash from the url string, and time is short for this demo, so to rush it, I just threw in the actual url as a string instead of relying on the config

Reply:
Don't we need the config for local proxying?

Reply:
Do we? When I throw make this line

`const url = `${await getOntologyUrl()}translate/paragraph`

it errors on `POST https://local.broadinstitute.org:3000/translate/paragraph 404 (Not Found)